driver/gaze_master_plugins/src/plugin.cpp

```diff
@@ -0,0 +1,69 @@
+#if defined(LINUX)
+	#define PLUGIN_DLL_EXPORT extern "C" __attribute__((visibility("default")))
+	#define PLUGIN_DLL_IMPORT extern "C"
+#elif defined(WIN)
+	#define PLUGIN_DLL_EXPORT extern "C" __declspec(dllexport)
+	#define PLUGIN_DLL_IMPORT extern "C" __declspec(dllimport)
+#else
+	#error "Unsupported Platform."
+#endif
+
+#include "gaze_master_plugin_interface.h"
+
+#include <fstream>
+
+class GazeLogger: public gaze::PluginBase {
+public:
+	GazeLogger() = default;
+	virtual ~GazeLogger() = default;
+
+	virtual bool Activate(const gaze::PluginVersion_t version) {
+		if (version.major != gaze::g_plugin_interface_version_major &&
+			version.minor != gaze::g_plugin_interface_version_minor &&
+			version.hotfix != gaze::g_plugin_interface_version_hotfix)
+		{
+			m_error_msg = "plugin interface version miss match";
+			return false; // interface version incompatibility is a no no
```

Comment:
Are we doing the boost plugin attempt in Driver Madness 8?

I think we should, since this has so many commits already. Just need to add it to the todo.